iml-agent/src/action_plugins/high_availability/fixtures/create-client-cloned.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="utf-8"?><clone id="cl-lustre-client"><primitive class="systemd" id="lustre-client" type="mnt-lustre.mount"><operations><op id="lustre-client-start-timeout-900s" name="start" timeout="900s" /><op id="lustre-client-monitor-timeout-60s" interval="60s" name="monitor" /></operations><meta_attributes id="lustre-client-meta_attributes"><nvpair id="lustre-client-meta_attributes-target-role" name="target-role" value="Stopped" /></meta_attributes></primitive></clone>
```

Comment:
format with `xmllint --format` ?

Reply:
no, then I can't do string matching, and elementtree does not implement Eq for Elements